forward/fwd.go

```diff
@@ -327,8 +327,10 @@ func (f *httpForwarder) serveWebSocket(w http.ResponseWriter, req *http.Request,
 	}
 	go replicate(targetConn, underlyingConn, "backend", "client")
 	go replicate(underlyingConn, targetConn, "client", "backend")
-	err = <-errc
-	log.Infof("vulcand/oxy/forward/websocket: websocket proxying complete: %v", err)
+	err = <-errc // One goroutine complete
+	log.Infof("vulcand/oxy/forward/websocket: > websocket proxying complete: %v", err)
+	err = <-errc // Both goroutines complete
+	log.Infof("vulcand/oxy/forward/websocket: < websocket proxying complete: %v", err)
```

Comment:
I reviewed the whole fine. Saying "first one of the bidirectional channel closed" and "second one of the bidirectional channel closed" should be sufficient. replicate() logs which direction channel it closed anyway.